tests/unit/core/data/entity/test_base.py

```diff
@@ -0,0 +1,46 @@
+# Copyright (C) 2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+"""Unit tests of base data entity."""
+
+import numpy as np
+import torch
+from otx.core.data.entity.base import ImageInfo, ImageType, OTXBatchDataEntity, OTXDataEntity
+from torchvision import tv_tensors
+
+
+class TestOTXDataEntity:
+    def test_image_type(self) -> None:
+        data_entity = OTXDataEntity(
```

Comment:
I recommend you to make `OTXDataEntity`, `OTXBatchDataEntity` and their variants to the pytest fixtures. So, they can be reused by other tests.

Reply:
Done by [06bf724](https://github.com/openvinotoolkit/training_extensions/pull/2711/commits/06bf7249cc4c8d47b5bec215c1fb3ef8d9929022)